terraform/datadog/router.tf

```diff
@@ -66,6 +66,22 @@ resource "datadog_monitor" "route_update_latency" {
   }
 }
 
+# This variable is used in the resource with same name to decide in which
+# environments it should be enabled or disable.
+#
+# The resource will refer this variable using a `lookup` function in the count
+# attribute.
+#
+# To enable the resource for a given environment, add an entry here.
+variable "__datadog_monitor_total__routes_drop__enabled" {
+  description = "Selector to enable/disable the named resource"
+
+  default = {
+    "dev"  = 1
+    "prod" = 1
```

Comment:
There was some discussion in Slack yesterday about moving this logic to the `tfvars` files where we already change some behaviour based on the AWS account. What was the conclusion?

Also, should the default be enabled, and we explicitly disable it for CI and staging? That would match the other flags we use.